File: samples/search/python/perform-search.py
import requests
from requests.auth import HTTPBasicAuth
import json

api_key = ""  # Your API key here, required
track_id = ""   # Track ID here, required
search_text = ""    # Your search text here
page = 0    # Page number
items = 50  # Number of items to show per page

url = "https://api-us.musiio.com/v1/search/perform"

payload = {
    "id": track_id,
    "text": search_text,
    "page": page,
    "items": items
}

headers = {
  'Content-Type': 'application/json'
}

response = requests.post(url, auth=HTTPBasicAuth(api_key, ''), headers=headers, data=json.dumps(payload))

print(response.text.encode('utf8'))

File: samples/search/js/perform-search.js
const API_KEY = "";     // Your API key here
const TRACK_ID = "";    // Track ID here
const SEARCH_TEXT = ""; // Search text input
const PAGE = 0;             // page number
const ITEMS_PER_PAGE = 50;  // items per page

var myHeaders = new Headers();
myHeaders.append("Content-Type", "application/json");
myHeaders.append("Authorization", "Basic " + btoa(API_KEY + ":"));

var raw = JSON.stringify({ 
    "id": TRACK_ID, 
    "text": SEARCH_TEXT, 
    "page": PAGE, 
    "items": ITEMS_PER_PAGE
});

var requestOptions = {
    method: 'POST',
    headers: myHeaders,
    body: raw,
    redirect: 'follow'
};

fetch("https://api-us.musiio.com/v1/search/perform", requestOptions)
    .then(response => response.text())
    .then(result => console.log(result))
    .catch(error => console.log('error', error));